tests/test_episodes.py

```diff
@@ -8,6 +8,26 @@ class EpisodeTestCase(TestCase):
 
   def setUp(self):
     super(EpisodeTestCase, self).setUp()
+    Recommendation.query.delete()
+    episodes_dao.clear_all_recommendations_counts()
+    db_session_commit()
+
+  def tearDown(self):
```

Comment:
Why is tearDown needed? setUp is also run before each test, so I think it should be fine without it. Same thing for the one in test_series

Reply:
I wanted to clean up after running the test so the next person writing a test doesn't need to worry about calling `setUp`.  I didn't call any recommendation deletion / clearing functions in `setUp` initially and my tests were failing b/c of residual info from other tests.  This shouldn't be the case.

Reply:
I should probably fix this then. I noticed that my db is not reset after running tests(I remember users (changeusername) for instance leaves one of the users as bob).

Reply:
We can make a separate PR regarding this, made an issue here: #142

Reply:
Hmm I feel like every test should also be responsible for calling setup anyways (and I think they are right now), when writing a test I would rather just ensure that the db is in the state I want to be, rather than trusting that the other tests are cleaning up for themselves

Reply:
that might be a lot to keep track of per test tbh.

Reply:
oh actually I see your point, didn't look at the tests closely enough, nevermind thats fair